src/js/stories/Button.stories.js

```diff
@@ -49,10 +54,15 @@ stories.add('Default', () => (
         <h2 className="h4 mb-2">Icon Only.</h2>
         <Button
           theme={select('Theme', themes, 'primary')}
-          icon={select('Icon', icons, 'media')}
           isDisabled={boolean('Disabled', false)}
           isOutline={boolean('Outline', false)}
           size={select('Size', sizes, 'md')}
+          icon={select('Icon', icons, 'media')}
+          iconTheme={
+            boolean('Show Icon', false) &&
+            boolean('Use theme?', false) &&
+            select('Icon Theme', themes, 'primary')
```

Comment:
I think we could just have `boolean('Show Icon', false) && select('Icon Theme', themes)`

Reply:
But then you can't go back to the text color after picking one of the themes. Otherwise, I can add a theme "none" if you prefer.

Reply:
Ok fair point.  Can we do a default of "white" then because primary theme icon on primary theme button disappears.